RELEASES.md

```diff
@@ -1,3 +1,47 @@
+# Version 1.0.0-alpha.2
+
+## Breaking Changes
+
+### `neon::object::This`
+
+https://github.com/neon-bindings/neon/pull/918
+
+Trait [`neon::object::This`](https://docs.rs/neon/latest/neon/object/trait.This.html) has been removed. `This` was primarily added for use with the `declare_types!` macro to generate classes. The macro was removed and `This` is no longer needed. Additionally, the `This` argument on `JsFunction` was found to be _invalid_ because it asserted at compile time a type for `this` that could change at runtime.
+
+**Note**: This is _not_ unsound because the type would be checked by Node-API and result in a `panic`.
```

Comment:
Since this is functionality we've removed, it's probably not worth a whole separate line and bold comment. I'd just put it in a parenthetical at the end of the previous paragraph and use the past tense:

```
...a type for `this` that could change at runtime. (Note that this
was _not_ unsound because the type would be checked by Node-API
and result in a `panic`.)
```